great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -57,7 +58,7 @@ class GeCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
         **{
             GeCloudRESTResource.BATCH: "batches",
             GeCloudRESTResource.CHECKPOINT: "checkpoints",
-            GeCloudRESTResource.CONTRACT: "contracts",
+            GeCloudRESTResource.CONTRACT: "contracts",  # Deprecated
```

Comment:
We need to keep this around as our E2E tests rely on old behavior - leaving a generic comment here but just wanted to flag for discussion before we move forward.